private/protocol/restjson/restjson.go

```diff
@@ -7,13 +7,13 @@ package restjson
 
 import (
 	"encoding/json"
+	"github.com/aws/aws-sdk-go-v2/private/protocol"
 	"io"
 	"io/ioutil"
 	"strings"
 
 	request "github.com/aws/aws-sdk-go-v2/aws"
 	"github.com/aws/aws-sdk-go-v2/aws/awserr"
-	"github.com/aws/aws-sdk-go-v2/private/protocol"
```

Comment:
Nit this change probalby isn't intended. running `goimports` to format the code should prevent things like this.